interfaces/Portalicious/src/environments/environment.ts

```diff
@@ -8,7 +8,7 @@ export const environment = {
   // Configuration/Feature-switches:
   defaultLocale: 'en',
   envName: '', // To highlight the environment used
-  locales: 'en,ar,fr,nl,es', // Comma-separated string of enabled locales (i.e: 'en,es,nl_BE'). Each should be available in: `./src/app/services/language.service.ts`
+  locales: 'en,nl', // Comma-separated string of enabled locales (i.e: 'en,es,nl_BE'). Each should be available in: `./src/locale`
```

Comment:
```suggestion
  locales: 'en,nl', // Comma-separated string of enabled locales. Each should be available in: `./src/locale`
```

To prevent some confusion about the 'format', i.e. useing `_` or `-` as a separator, etc..